sklearn/utils/fixes.py

```diff
@@ -25,6 +25,7 @@
 
 np_version = parse_version(np.__version__)
 sp_version = parse_version(scipy.__version__)
+sp_base_version = parse_version(sp_version.base_version)
```

Comment:
For our purposes, I think we should always use the base version. What do you think of setting `np_version` and `sp_version` to their base versions?

Reply:
Yep almost convinced as well. I will check that we don't have corner cases relying on bugfix versions in our codebase.

Reply:
Do you want to do that as part of this PR? I think it should be done a follow PR to keep things focused on fixing one problem at a time.

Reply:
I'm okay with having another `sp_base_version`. I was concerned with adding another public `*_version` variable that could confuse our contributors or third party developers that depend on `utils.fixes.sp_version`. (They need to decide which one to use)